Analysis/config/2016/sources.cfg

```diff
@@ -7,7 +7,7 @@ qcd_method: invert_tau
 unc_sources: Central None
 unc_sources: JES JetReduced_Total JetReduced_Absolute JetReduced_Absolute_year JetReduced_BBEC1 JetReduced_BBEC1_year JetReduced_EC2 JetReduced_EC2_year JetReduced_FlavorQCD JetReduced_HF JetReduced_HF_year JetReduced_RelativeBal JetReduced_RelativeSample_year
 unc_sources: LES TauES TauES_DM0 TauES_DM1 TauES_DM10 TauES_DM11 EleFakingTauES_DM0 EleFakingTauES_DM1 MuFakingTauES
-norm_unc_sources: EleTriggerUnc MuonTriggerUnc TauTriggerUnc_DM0 TauTriggerUnc_DM1 TauTriggerUnc_DM10 TauTriggerUnc_DM11 TauVSjetSF_DM0 TauVSjetSF_DM1 TauVSjetSF_3prong TauVSjetSF_pt20to25 TauVSjetSF_pt25to30 TauVSjetSF_pt30to35 TauVSjetSF_pt35to40 TauVSjetSF_ptgt40 TauVSeSF_barrel TauVSeSF_endcap TauVSmuSF_etaLt0p4 TauVSmuSF_eta0p4to0p8 TauVSmuSF_eta0p8to1p2 TauVSmuSF_eta1p2to1p7 TauVSmuSF_etaGt1p7 EleIdIsoUnc MuonIdIsoUnc TopPt L1_prefiring PileUp PileUpJetId_eff PileUpJetId_mistag TauCustomSF_DM0 TauCustomSF_DM1 TauCustomSF_DM10 TauCustomSF_DM11
+norm_unc_sources: EleTriggerUnc MuonTriggerUnc TauTriggerUnc_DM0 TauTriggerUnc_DM1 TauTriggerUnc_DM10 TauTriggerUnc_DM11 TauVSjetSF_DM0 TauVSjetSF_DM1 TauVSjetSF_3prong TauVSjetSF_pt20to25 TauVSjetSF_pt25to30 TauVSjetSF_pt30to35 TauVSjetSF_pt35to40 TauVSjetSF_ptgt40 TauVSeSF_barrel TauVSeSF_endcap TauVSmuSF_etaLt0p4 TauVSmuSF_eta0p4to0p8 TauVSmuSF_eta0p8to1p2 TauVSmuSF_eta1p2to1p7 TauVSmuSF_etaGt1p7 EleIdIsoUnc MuonIdIsoUnc TopPt L1_prefiring PileUp PileUpJetId_eff PileUpJetId_mistag btag_lf btag_hf btag_hfstats1 btag_hfstats2 btag_lfstats1 btag_lfstats2 btag_cferr1 btag_cferr2 TauCustomSF_DM0 TauCustomSF_DM1 TauCustomSF_DM10 TauCustomSF_DM11
```

Comment:
Sorry, I forgot to mention new trigger unc in the last comment. Please, check that all newly introduced unc are in the list.